src/common/Collision/Management/VMapManager2.cpp

```diff
@@ -166,7 +166,7 @@ namespace VMAP
     bool VMapManager2::isInLineOfSight(unsigned int mapId, float x1, float y1, float z1, float x2, float y2, float z2)
     {
 #if defined(ENABLE_VMAP_CHECKS)
-        if (!isLineOfSightCalcEnabled() || !IsVMAPDisabledForPtr(mapId, VMAP_DISABLE_LOS))
+        if (!isLineOfSightCalcEnabled() || IsVMAPDisabledForPtr(mapId, VMAP_DISABLE_LOS))
```

Comment:
how is this change related to the title of this PR?

Reply:
`DisableMgr::IsDisabledFor` to `IsVMAPDisabledForPtr` ?

Reply:
General change - delete `game-interface` from `common` lib
![image](https://user-images.githubusercontent.com/18329778/115876732-e0382380-a470-11eb-8ede-9dc2cc3c8562.png)

Other changes - support build without game lib

Reply:
I meant why removing `!` from `!IsVMAPDisabledForPtr`

Reply:
Look all changes